docs/competition/matches/match-scoring.md

```diff
@@ -0,0 +1,57 @@
+
+<sup>1</sup>: The score sheets currently label the pink corner as purple. This isn't a major issue because they both start with the letter P, but it's something to be aware of.
+
+As well as scoring, your responsibilities include making sure robots start in the right corner and switching off robots that do not automatically turn off after a match ends or when a team requests that you do so. You must also make sure that any arena props (tokens, for example) are in the correct place at the start of each match as detailed below. You are not responsible, however, for making sure a match starts with the correct number of teams -- that is down to the shepherds.
+
+Teams must not be allowed in the arena during a match and only one team member is allowed in the area between the crowd barrier and arena wall. It is the responsibility of the team members to place their robot in the start zone before the match starts. If a robot moves off before the match starts (aside from minor servo-resetting arm movements) teams are allowed to switch off their robot, reset it and start again. However, if the mis-started robot touches any arena prop then they are immediately disqualified. Once the robot start button has been pressed and the match has started, teams are not allowed to interact with their robots until after scoring has taken place.
+
+If the team members indicates that they wish for you to turn off their robot (i.e. "throwing in the towel") you may do so but by doing so they forfeit the match which will be recorded in the score sheets as a disqualification. There are two cases where you are allowed to switch off a robot yourself, one is when the match ends and the other is when a robot is causing **unintentional and significant** damage to another robot (i.e. mashes into another robot, pins it, stopping it from being able to function -- includes one dragging the other around if they become intertwined), and in that case it is up to you to decide which robot should be switched off but their match is not forfeited. If the damage is intentional, then the team will get forfeited (potentially for previous matched too), but this decision should be left to the judge. A checklist for robot damage is available below.
+
+You must read [points 1.1 to 1.7 and 3.5 of the rules](https://www.studentrobotics.org/docs/rules/) before starting this role to make sure you understand the game, the scoring rules and how the arena is laid out.
```

Comment:
Probably worth specifying that they should read the whole game-specific section of the rules. Documenting the exact points isn't useful as it'll change between the years

Reply:
14d1dca